NAS-102193: Update note about edit/delete TrueNAS HA interfaces (#1452)
freenas/freenas-docs: userguide/network.rst

:guilabel:`Type`.

#ifdef truenas

> .. note:: The ability to edit or delete interfaces is disabled when

Reviewer: Simplify:
```suggestion
.. note:: Interfaces cannot be edited or deleted when
```

>    :ref:`High Availability (HA) <Failover>` has been configured and

Reviewer: Also can be simplified, since "enabled" is a superset of "configured":
```suggestion
   :ref:`High Availability (HA) <Failover>` has been
```
